packages/modal/src/ModalView.tsx

```diff
@@ -1,172 +1,29 @@
-]);
-
-const defaultHorizontalSpacing = 18;
-const defaultVerticalSpacing = 64;
-
-const backdrop = css`
-  overflow-y: auto;
-  position: fixed;
-  top: 0;
-  left: 0;
-  right: 0;
-  bottom: 0;
-  opacity: 0;
-  transition: opacity ${transitionDuration.default}ms ease-in-out;
-`;
-
-const visibleBackdrop = css`
-  opacity: 1;
-`;
-
-const scrollContainer = css`
-  position: absolute;
-  min-height: 100%;
-  width: 100%;
-  padding: ${defaultVerticalSpacing}px ${defaultHorizontalSpacing}px;
-  overflow-y: auto;
-  display: flex;
-  align-items: center;
-  justify-content: center;
-`;
-
-const modalContentStyle = css`
-  transition: all ${transitionDuration.default}ms ease-in-out;
-  margin: auto;
-  max-height: calc(100% - ${defaultVerticalSpacing}px);
-  position: relative;
-  pointer-events: all;
-  transform: translate3d(0, -16px, 0);
-  opacity: 0;
-
-  &:focus {
-    outline: none;
-  }
-`;
-
-const modeStyles = css`
-  color: ${uiColors.gray.dark3};
-  background-color: ${uiColors.white};
-  font-family: ${fontFamilies.default};
-  border-radius: 24px;
-  padding: 40px 36px;
-  box-shadow: 0px 8px 20px -8px ${transparentize(0.4, palette.black)};
-`;
-
-const visibleModalContentStyle = css`
-  transform: translate3d(0, 0, 0);
-  opacity: 1;
-`;
-
-const modalSizes: { readonly [K in ModalSize]: string } = {
-  small: css`
-    width: 400px;
-  `,
-
-  default: css`
-    width: 600px;
-  `,
-
-  large: css`
-    ${mq({
-      width: ['720px', '720px', '960px'],
-    })}
-  `,
-};
-
-const baseCloseButtonStyles = css`
-  position: absolute;
-  cursor: pointer;
-`;
-
-const closeButton: Record<
-  Mode,
-  Record<CloseIconColor, string> & Record<'position', string>
-> = {
-  [Mode.Light]: {
-    [CloseIconColor.Default]: css`
-      color: ${palette.gray.base};
-    `,
-    [CloseIconColor.Dark]: css`
-      color: ${palette.gray.dark1};
-    `,
-    [CloseIconColor.Light]: css`
-      color: ${palette.white};
-    `,
-    position: css`
-      // x-icon should be 24px from edge. IconButton is 28x28 and Icon is 16x16
-      // so there's already (28 - 16) / 2 = 6px of spacing. 24 - 6 = 18.
-      right: 18px;
-      top: 18px;
-    `,
-  },
-  [Mode.Dark]: {
-    [CloseIconColor.Default]: css`
-      color: ${uiColors.gray.base};
-
-      &:hover {
-        color: ${uiColors.gray.base};
-      }
-    `,
-    [CloseIconColor.Dark]: css`
-      color: ${uiColors.gray.base};
-
-      &:hover {
-        color: ${uiColors.gray.base};
-      }
-    `,
-    [CloseIconColor.Light]: css`
-      color: ${uiColors.gray.base};
-
-      &:hover {
-        color: ${uiColors.gray.base};
-      }
-    `,
-    position: css`
-      // x-icon should be 16px from edge. IconButton is 28x28 and Icon is 16x16
-      // so there's already (28 - 16) / 2 = 6px of spacing. 16 - 6 = 10.
-      right: 10px;
-      top: 10px;
-    `,
-  },
-};
-
-const closeClassName = createUniqueClassName();
+import { CloseIconColor, ModalProps, ModalSize } from '././types';
+import {
```

Comment:
nit, non-blocking, curious about our thoughts on doing import * from './styles' so we dont have to name them all out

Reply:
cc @spark33 what do you think?

Reply:
I like that idea! Especially since we're having a `styles.ts` for every subdirectory, all styles exported from that file should be relevant in the component file.

Reply:
yeah I don't mind

Reply:
Looks like we would have to use this syntax if we import `*`, `import * as styles from './Modal.styles';`. So for each style we would have to use individual exports,`style.modalContentStyle`

https://bobbyhadz.com/blog/typescript-import-all-exports-from-file

If we have to do this I'd rather name all the exports at the top. Thoughts?

Reply:
yeah you're probably right. feel free to leave as-is